src/com/fediroryshchuk/homework7/brokesinglton/singletonpattern/PoliticianSelection.java

```diff
@@ -0,0 +1,47 @@
+package com.fediroryshchuk.homework7.brokesinglton.singletonpattern;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class PoliticianSelection {
+    public static SeniorGovernmentOfficials senior;
+
+    static {
+        try {
+            callPolitician();
+
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void callPolitician() throws IOException {
+
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        String s = null;
+        try {
+            s = reader.readLine();
+
+            switch (s) {
+                case "Порошенко":
+                    senior = President.getInstance();
+                    System.out.println("He is president");
+                    break;
+                case "Гройсман":
+                    senior = PrimeMinister.getInstance();
+                    System.out.println("He is prime-minister");
+                    break;
+                case "Парубий":
+                    senior = Speaker.getInstance();
+                    System.out.println("He is speaker");
+                    break;
+                default: System.out.println("Unknown person. Try again...");
+
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    reader.close();
```

Comment:
You should use try-finally block or try with resources.
1. https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
![image](https://user-images.githubusercontent.com/20267148/54507911-31998e80-494c-11e9-9217-3729372e1d52.png)
2. 
![image](https://user-images.githubusercontent.com/20267148/54507934-44ac5e80-494c-11e9-8a81-a7745a463887.png)